### PR TITLE
Fix quick start bug within the awsxrayreceiver

### PR DESCRIPTION
**Pull request by humivo**
**Description:** Fix bug within the awsxrayreceiver where the start func was not called.

### CODE REVIEW

**Review by bogdandrutu — APPROVED** [review]

File: receiver/awsxrayreceiver/receiver.go [hunk 1]
```diff
@@ -117,6 +117,7 @@ func (x *xrayReceiver) Shutdown(_ context.Context) error {
 func (x *xrayReceiver) start() {
 	incomingSegments := x.poller.SegmentsChan()
 	for seg := range incomingSegments {
+		x.obsrecv.StartTraceDataReceiveOp(seg.Ctx)
```
Comment (bogdandrutu) [inline receiver/awsxrayreceiver/receiver.go:120]: This returns a context that should be used in the equivalent End.

**Review by bogdandrutu — CHANGES_REQUESTED** [review]

**Comment by bogdandrutu** [comment]
@anuraaga @Aneurysm9 please review

File: receiver/awsxrayreceiver/receiver.go (hunk 1 above)
Comment (humivo) [reply receiver/awsxrayreceiver/receiver.go]: Edited in latest commit

**Review by mxiamxia — APPROVED** [review]
LGTM!

**Review by anuraaga — APPROVED** [review]

File: receiver/awsxrayreceiver/receiver.go [hunk 2]
```diff
@@ -117,6 +117,7 @@ func (x *xrayReceiver) Shutdown(_ context.Context) error {
 func (x *xrayReceiver) start() {
 	incomingSegments := x.poller.SegmentsChan()
 	for seg := range incomingSegments {
+		seg.Ctx = x.obsrecv.StartTraceDataReceiveOp(seg.Ctx)
```
Comment (bogdandrutu) [inline receiver/awsxrayreceiver/receiver.go:120]: `ctx := x.obsrecv.StartTraceDataReceiveOp(seg.Ctx)`

Every operation starts from `seg.Ctx` not from previous returned.

Comment (humivo) [reply receiver/awsxrayreceiver/receiver.go]: Changed in the latest commit

**Review by Aneurysm9 — APPROVED** [review]

**Review by bogdandrutu — APPROVED** [review]

**Pull request merged**